src/components/Resource/ResourceList.tsx

```diff
@@ -48,15 +48,125 @@ export default function ListView() {
   const showResourceCardList = (data: ResourceModel[]) => {
     if (data && !data.length) {
       return (
-        <div className="mt-64 flex flex-1 justify-center text-secondary-600">
+        <div className="w-full mt-64 flex flex-1 justify-center text-secondary-600">
           {t("no_results_found")}
         </div>
       );
     }
 
-    return data.map((resource, i) => (
-      <div key={i} className="rounded-lg border border-secondary-300 bg-white">
-        <ResourceBlock resource={resource} />
+    return data.map((resource: ResourceModel, i) => (
+      <div key={i} className="w-full border-b-2 border-gray-100 col-span-6">
+        <div className="border-3  flex grid w-full gap-1 overflow-hidden bg-white p-4 shadow sm:grid-cols-1 md:grid-cols-1 lg:grid-cols-5">
```

Comment:
We are moving away from shadows in our designs, lets use borders. `gray-300` or `gray-200` border would be fine